src/client.spec.ts

```diff
@@ -38,6 +38,9 @@ describe('ReactSDKClient', () => {
       getForcedVariation: jest.fn(() => null),
       getFeatureVariableBoolean: jest.fn(() => null),
       getFeatureVariableDouble: jest.fn(() => null),
+      getFeatureVariableJSON: jest.fn(() => null),
+      getAllFeatureVariables: jest.fn(() => { return {} }),
```

Comment:
Oh, interesting that `getAllFeatureVariables` was not here before. I guess it should be added, but does it affect any test?